src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -140,18 +150,51 @@ public static String getIndividualLastName(Entity entity) {
         return entity.getName().getLastName();
     }
 
-    public static String getIndividualPreferredHearingChannel(String hearingType, HearingSubtype hearingSubtype) {
-        // TODO Depends on SSCS-10273 - Needs to implement for Reference data of valid Hearing Channel codes
-        return null;
+    public static Optional<String> getIndividualPreferredHearingChannel(String hearingType,
+                                                                        HearingSubtype hearingSubtype,
+                                                                        HearingOptions hearingOptions) {
+        if (hearingType == null || hearingSubtype == null) {
+            return Optional.empty();
+        }
+
+        return shouldPreferNotAttendingHearingChannel(hearingType, hearingOptions) ? Optional.ofNullable(NOT_ATTENDING.getHmcReference())
+            : isYes(hearingSubtype.getWantsHearingTypeFaceToFace()) ? Optional.ofNullable(FACE_TO_FACE.getHmcReference())
+            : shouldPreferVideoHearingChannel(hearingSubtype) ? Optional.ofNullable(VIDEO.getHmcReference())
+            : shouldPreferTelephoneHearingChannel(hearingSubtype) ? Optional.ofNullable(TELEPHONE.getHmcReference())
+            : Optional.empty();
     }
 
-    public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions) {
-        // TODO Depends on SSCS-10273 - Needs to implement for Reference data to convert from SSCS Languages/Sign Languages to Reference languages
-        // if (isYes(hearingOptions.getLanguageInterpreter())) {
-        //     String signLanguageType = hearingOptions.getSignLanguageType();
-        //     String languages = hearingOptions.getLanguages();
-        // }
-        return null;
+    private static boolean shouldPreferNotAttendingHearingChannel(String hearingType, HearingOptions hearingOptions) {
+        return HEARING_TYPE_PAPER.equals(hearingType) || !hearingOptions.isWantsToAttendHearing();
+    }
+
+    private static boolean shouldPreferTelephoneHearingChannel(HearingSubtype hearingSubtype) {
+        return isYes(hearingSubtype.getWantsHearingTypeTelephone()) && nonNull(hearingSubtype.getHearingTelephoneNumber());
+    }
+
+    private static boolean shouldPreferVideoHearingChannel(HearingSubtype hearingSubtype) {
+        return isYes(hearingSubtype.getWantsHearingTypeVideo())
+            && nonNull(hearingSubtype.getHearingVideoEmail());
+    }
+
+    public static Optional<String> getIndividualInterpreterLanguage(HearingOptions hearingOptions) {
+        if (isTrue(hearingOptions.wantsSignLanguageInterpreter())) {
+            return getSignLanguage(hearingOptions)
+                .map(SignLanguage::getHmcReference);
```

Comment:
if getSignLanguage returns null or an invalid string, should it throw an error or continue to check to see if getInterpreterLanguage is valid before returning null?